apps/dotcom/client/src/tla/components/dialogs/TlaSignInDialog.tsx

```diff
@@ -177,22 +194,20 @@ function TlaEnterEmailStep({
 				<br />
 				<F defaultMessage="Create a free account to save your work, collaborate in real-time, and more." />
 			</div>
-			<SignIn.Root routing="virtual">
-				<SignIn.Step name="start">
-					<div className={styles.authGoogleButtonWrapper}>
-						{/* @ts-ignore this is fine */}
-						<Clerk.Connection name="google" asChild>
-							<TlaCtaButton
-								data-testid="tla-google-sign-in-button"
-								className={styles.authCtaButton}
-							>
-								<Clerk.Icon icon="google" />
-								<F defaultMessage="Sign in with Google" />
-							</TlaCtaButton>
-						</Clerk.Connection>
-					</div>
-				</SignIn.Step>
-			</SignIn.Root>
+			<div className={styles.authGoogleButtonWrapper}>
+				<TlaCtaButton
+					data-testid="tla-google-sign-in-button"
+					className={styles.authCtaButton}
+					onClick={handleGoogleSignIn}
+				>
+					<img
+						src="https://img.clerk.com/static/google.svg"
```

Comment:
Unlikely that this will change, but we could maybe still use clerk icon here to be on the safe side?